### ISSUE TITLE
If it is legal to reappear it in another extension?

### ISSUE THREAD

**Issue by GiacomoZheng** [aaron-bond/better-comments#113]
`better-comments` is such a useful extension and I have used it for a long time, but I created a new language this year and it not supported. At first I did some change on the source files of this extension, but it is not very convenient and allowable. 

Therefore, I add the something into the `tmlanguage.json`, like `comment.line.pure.*`, `commet.line.todo.*`. Meanwhile, I created a `theme` which can colorize these tabs.

In conclusion, I realize the function of `better-comments` with a totally different way.

@aaron-bond I want to know what's your attitude about it. Is it allowed?

I haven't published the newest version on `vscode marketplace` up to now, but I have uploaded it on my github.

Thanks.

**Comment by aaron-bond** [comment]
The extension and source code are released under MIT License. That means you're free to use it however you'd like :)

That being said, if you want some help getting your new language working with the better-comments extension instead of making a brand new one, I'm sure I could help?

**Comment by GiacomoZheng** [comment]
Thanks. It is better if this extesion `better-comments` can support my language. My language named `gm`, which can be found both in `vscode marketplace` and my `Github`.

While, in fact, up to now, I'm the only user of my language (although there is 40 times download, I don't konw who they are, and in fact the tutorials of it is written). It is true that I'll keep developing and popularizing it, but it may take a long for me to achieve it.

If you are still willing to help me, you can continue.

The line comment symbol is `;`, which is the same as the `lisp` and `racket`. You can just ignore the block comment of mine.

I'll keep the `gm - dark+` theme for myself as I think it is lighter and faster, but I'll stop the develop the other themes.

Thanks
